core/src/components/cat-form-hint/cat-form-hint.tsx

```diff
@@ -31,7 +31,7 @@ export const CatFormHint: FunctionalComponent<CatFormHintProps> = props => {
     <div id={id + '-hint'} class="hint-section">
       {errors.length
         ? errors.map(([key, params]) => (
-            <p class={(props.class ?? '') + ' input-hint cat-text-danger'}>{i18n.t(`error.${key}`, params)}</p>
+            <p tabindex="0" role="alert" class={(props.class ?? '') + ' input-hint cat-text-danger'}>{i18n.t(`error.${key}`, params)}</p>
```

Comment:
this is not an interactive element so it should not be tabable, also i think role=alert does not work with this dynamically created element. aria-live is only triggered for an element that changes it's content not on new elements (see hint box here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) 

Additionally this does not solve the described by issue that is only added to the input when there is a hint but should also be added when there is an error.

Reply:
your comment is resolved, now input errors support live updates in voice over